plugins/jobs/web_client/views/JobListWidget.js

```diff
@@ -1,190 +0,0 @@
-import _ from 'underscore';
```

Comment:
I must have missed this before, but we need to move this file back to its original name and location so that we don't break backward compatibility. If someone was importing this file into their own bundle, this would break their build.